pkg/config/signer_test.go

```diff
@@ -25,21 +25,21 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var publicKey, privateKey []byte
-
 func TestMain(m *testing.M) {
-	publicKey, privateKey = generatePublicAndPrivateKey()
-
 	os.Exit(m.Run())
 }
 
-func TestNewSigner(t *testing.T) {
+func TestNewSigningIdentity(t *testing.T) {
+	t.Parallel()
+
+	publicKey, privateKey := generatePublicAndPrivateKey()
+
 	t.Run("success", func(t *testing.T) {
 		gt := NewGomegaWithT(t)
-		signer, err := NewSigner([]byte(publicKey), []byte(privateKey), "test-msp")
+		signingIdentity, err := NewSigningIdentity([]byte(publicKey), []byte(privateKey), "test-msp")
```

Comment:
`publicKey` and `privateKey` are already byte slices, you don't need to cast them again

Reply:
done